docs/fsharp/tutorials/getting-started/getting-started-vscode.md

```diff
@@ -1,4 +1,264 @@
+
+![](media/getting-started-vscode/vscode-new-dir.png)
+
+Here you can choose to create the project in a specific directory.  If you leave it blank, it will choose the current directory Visual Studio Code is opened in.  This article uses the current directory.  Once you've made your decision, hit **Enter** to bring up the final stage:
+
+![](media/getting-started-vscode/vscode-proj-name.png)
+
+This lets you name your project.  F# uses PascalCase for project names.  This article uses `ClassLibrarySample` as the name.  Once you've entered the name you want for your project, hit **Enter**.
+
+If you followed these steps, you should get the Explorer on the left-hand side to look something like this:
+
+![](media/getting-started-vscode/vscode-new-proj-explorer.png)
+
+This template generates a few things you'll find useful:
+
+1. The F# project itself, underneath the `ClassLibraryDemo` folder.
+2. The correct directory structure for adding packages via [`Paket`](http://fsprojects.github.io/Paket/).
+3. A cross-platform build script with [`FAKE`](http://fsharp.github.io/FAKE/).
+4. The `paket.exe` executable which can fetch packages and resolve dependencies for you.
+5. A useful `.gitignore` file if you wish to add this project to Git-based source control.
+
+## Writing some code
+
+Open the `ClassLibraryDemo` folder.  You'll notice a few files:
+
+1. `ClassLibraryDemo.fs`, an F# source file with a class defined.
+2. `CassLibraryDemo.fsproj`, an F# project file used to build this project.
+3. `Script.fsx`, an F# script file which loads the source file.
+4. `paket.references`, a Paket file which specifies the project dependencies.
+
+Open `Script.fsx`, and add the following code at the end of it:
+
+[!code-fsharp[ToPigLatin](../../../../samples/snippets/fsharp/getting-started/to-pig-latin.fsx)]
+
+This function converts a word to a form of [Pig Latin](https://en.wikipedia.org/wiki/Pig_Latin).  Don't worry about the details of the code - that will be covered below.  The next step is to execute it using F# Interactive (FSI).
+
+Highlight the entire function (it should be 11 lines long).  Once it is highlighted, hold the **alt** key and hit **Enter**.  You'll notice a Window pop up below, and it should show something like this:
+
+![](media/getting-started-vscode/vscode-fsi-new.png)
+
+This did three things:
+
+1. It started the FSI process.
+2. It sent the code you highlighted over the FSI process.
+3. The FSI process evaluated the code you sent over.
+
+Because what you sent over was a [function](../../language-reference/functions/index.md), you can now call that function with FSI!  In the interactive window, type the following:
+
+```fsharp
+toPigLatin "banana";;
+```
+
+You should see the following result:
+
+```fsharp
+val it : string = "ananabay"
+```
+
+Hooray!  It works when the first character isn't a vowel.  What about when it's a vowel?  Enter the following:
```

Comment:
Hooray seems very informal to me. Maybe change this paragraph to:
"Now, let's try with a vowel as the first letter. Enter the following:"